app/src/main/java/ru/prometeydev/movie/model/domain/MovieDto.kt

```diff
@@ -0,0 +1,17 @@
+package ru.prometeydev.movie.model.domain
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class MovieDto(
```

Comment:
What is difference between movie dto and movie details dto?

Reply:
I think you can make a general class. Just a query that returns a list of popular films in an element of one movie stores truncated information, and a query that returns movie details by id contains a more extensive set of data. Therefore, I passed two different classes. But I think you can limit yourself to one class.

Reply:
Up to you, but I would probably use one class